### ISSUE TITLE
Statistics screen for the engine

### ISSUE THREAD

**Issue by dballona** [dballona/actionmessage#14]
As described on #13 

```
# on routes.rb
mount ActionMessage::Statistics => '/sms'
```

This engine would define two routes, one for hooks for the adapters to post messages into (such as Twilio). and the other one for statistics;

we’d probably have a module to include on the tracker model under the app for Mongoid, and a generator for SQL databases (generates both migration and model);

and on the statistics action, we’d show statistics that are common between adapters, such as delivery status, amount sent by date, amount received.

Basically consists on building something next to Sidekiq::Web engine on this interface.
